### PR TITLE
STYLE: Extending codespell to pandas/tests Part 2

### PR DESCRIPTION
**Pull request by 01-vyom**
- [X] xref #38802 
- [X] tests added / passed
- [X] Ensure all linting tests pass, see [here](https://pandas.pydata.org/pandas-docs/dev/development/contributing.html#code-standards) for how to run them
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/tests/frame/indexing/test_setitem.py [hunk 1]
```diff
@@ -433,13 +433,13 @@ def test_setitem_intervals(self):
         assert is_categorical_dtype(df["D"].dtype)
         assert is_interval_dtype(df["D"].cat.categories)
 
-        # Thes goes through the Series constructor and so get inferred back
+        # These goes through the Series constructor and so get inferred back
```
Comment (dsaxton) [inline pandas/tests/frame/indexing/test_setitem.py:436]: "These go" most likely?

**Review by jreback — COMMENTED** [review]
lgtm ex @dsaxton comment

**Comment by jreback** [comment]
doesn't this have to turn on this in the codespell config? (or is this a precursor)

**Comment by 01-vyom** [comment]
> doesn't this have to turn on this in the codespell config? (or is this a precursor)

Yes @jreback, by removing the exclude from the codespell config, we get 68 file changes so we decided to split those changes into 3 PRs and include the codespell config change in the final PR in-order to pass the pre-commit for all the PRs.

More Info here: #40320

**Comment by 01-vyom** [comment]
Made all the necessary changes.

**Comment by MarcoGorelli** [comment]
Thanks @01-vyom !

**Pull request merged**